<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialogproduit</class>
 <widget class="QMainWindow" name="dialogproduit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 255, 127)</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Gestion des commandes</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>330</y>
        <width>301</width>
        <height>211</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>Gestion</string>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>261</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Sélectionnez un element dans le tableau</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_edit">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>60</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Modifier</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_supp">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>90</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Supprimer</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_7">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>160</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Ficher D'historique</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_pdf_produit">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>130</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>PDF</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>301</width>
        <height>301</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>Ajouter une commande</string>
      </property>
      <widget class="QPushButton" name="pushButton_ajouter">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>250</y>
         <width>91</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget_6">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>81</width>
         <height>211</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Désignation</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_15">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Réf</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_11">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Qté</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Prix Unit</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Montant</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget_7">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>30</y>
         <width>169</width>
         <height>211</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QLineEdit" name="lineEdit_des">
          <property name="maxLength">
           <number>12</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_ref">
          <property name="maxLength">
           <number>9</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_qte">
          <property name="inputMethodHints">
           <set>Qt::ImhNone</set>
          </property>
          <property name="maxLength">
           <number>3</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_pr">
          <property name="maxLength">
           <number>9</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_montant">
          <property name="maxLength">
           <number>6</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>160</y>
        <width>451</width>
        <height>371</height>
       </rect>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>20</y>
        <width>441</width>
        <height>131</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>Recherche et tri</string>
      </property>
      <widget class="QPushButton" name="pushButton_rech">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>80</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Recherche</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_search">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>40</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>50</y>
         <width>31</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Réf</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_trie">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>90</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Trier</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_montant">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>60</y>
         <width>101</width>
         <height>18</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>7</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Par montant </string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_qte">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>40</y>
         <width>91</width>
         <height>18</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>7</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Par quantite</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_az">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>20</y>
         <width>83</width>
         <height>18</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>7</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Tri A à Z</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
